pkg/printers/internalversion/describe_test.go

```diff
@@ -1398,3 +1399,61 @@ URL:	http://localhost
 		}
 	}
 }
+
+func TestDescribePodSecurityPolicy(t *testing.T) {
+	expected := []string{
+		"Name:.*mypsp",
+		"Allow Privileged:.*false",
+		"Default Add Capabilities:.*<none>",
+		"Required Drop Capabilities:.*<none>",
+		"Allowed Capabilities:.*<none>",
+		"Allowed Volume Types:.*<none>",
+		"Allow Host Network:.*false",
+		"Allow Host Ports:.*<none>",
+		"Allow Host PID:.*false",
+		"Allow Host IPC:.*false",
+		"Read Only Root Filesystem:.*false",
+		"SELinux Context Strategy: RunAsAny",
+		"User:.*<none>",
+		"Role:.*<none>",
+		"Type:.*<none>",
+		"Level:.*<none>",
+		"Run As User Strategy: RunAsAny",
+		"FSGroup Strategy: RunAsAny",
+		"Supplemental Groups Strategy: RunAsAny",
+		"Ranges:.*<none>",
```

Comment:
I wouldn't check for Ranges because there are more than one Range and when we change any of them, test is starting to fail.